apps/teams-test-app/src/components/privateApis/CopilotAPIs.tsx

```diff
@@ -23,11 +23,34 @@ const CopilotAPIs = (): ReactElement => {
       },
     });
 
+  const SendCustomTelemetryData = (): ReactElement =>
+    ApiWithTextInput<{
+      name: copilot.Stage;
```

Comment:
nit: how do you think of copilot.Telemetry or copilot.Telemetry.Stage?

It feels copilot.Stage is not obvious enough to be correlated with telemetry by looking at the test app perspective.